DiffPlex/Chunkers/CustomFunctionChunker.cs

```diff
@@ -8,8 +8,7 @@ public class CustomFunctionChunker: IChunker
 
         public CustomFunctionChunker(Func<string, string[]> customChunkerFunc)
         {
-            if(customChunkerFunc == null) throw new ArgumentNullException(nameof(customChunkerFunc));
-            this.customChunkerFunc = customChunkerFunc;
+            this.customChunkerFunc = customChunkerFunc ?? throw new ArgumentNullException(nameof(customChunkerFunc));
```

Comment:
I honestly don't love this syntax. I rather keep this the other way

Reply:
Alright, I will rollback this.